path_oidc.go

```diff
@@ -308,7 +387,33 @@ func (b *jwtAuthBackend) authURL(ctx context.Context, req *logical.Request, d *f
 		return resp, nil
 	}
 
-	resp.Data["auth_url"] = oauth2Config.AuthCodeURL(stateID, oidc.Nonce(nonce))
+	authCodeOpts := []oauth2.AuthCodeOption{
+		oidc.Nonce(nonce),
+	}
+
+	// Add "form_post" param if requested. Note: the operator is allowed to configure "query"
+	// as well, but that is the default for the AuthCode method and needn't be explicitly added.
+	if config.OIDCResponseMode == responseModeFormPost {
+		authCodeOpts = append(authCodeOpts, oauth2.SetAuthURLParam("response_mode", "form_post"))
```

Comment:
nit Worth setting it to the const value here?

Reply:
👍